src/icalendar/prop/__init__.py

```diff
@@ -808,15 +808,53 @@ def from_ical(ical):
 
 
 class vDatetime(TimeBase):
-    """Render and generates icalendar datetime format.
-
-    vDatetime is timezone aware and uses a timezone library.
-    When a vDatetime object is created from an
-    ical string, you can pass a valid timezone identifier. When a
-    vDatetime object is created from a python datetime object, it uses the
-    tzinfo component, if present. Otherwise a timezone-naive object is
-    created. Be aware that there are certain limitations with timezone naive
-    DATE-TIME components in the icalendar standard.
+    """Date-Time
```

Comment:
Please take a look at how this renders at https://icalendar--917.org.readthedocs.build/en/917/api.html#icalendar.prop.vDatetime. You'll see it doesn't follow the patterns of existing property docstrings, such as https://icalendar--917.org.readthedocs.build/en/917/api.html#icalendar.prop.vDate. Please build the docs locally to review and test before pushing commits to correct the display issues. Thank you!

Reply:
This is why I rather merge 2 or more PRs instead of one: http://hintjens.com/blog:106

Reply:
@stevepiercy reversion is definitely against my intent. I mean that it was premature to merge this PR, and doing so deprived me the opportunity to help or mentor the contributor to correct the bug, and for the contributor to improve and learn from it. A merge says "this is done, move along".

Since you merged it, ignoring my concerns, and after handing off the responsibility to me in https://github.com/collective/icalendar/pull/917#issuecomment-3421177497, I think it's your responsibility to make it right.